constants/authorization.js

```diff
@@ -1,24 +1,19 @@
 // Utilities
 import Router from "next/router";
 
-export const google_authorization = {
-  key: "covid19to.api.token",
+const google_authorization = {
+  key: "covid19to.api.token"
 };
 
-export const is_authorized = () => {
-  if (typeof window !== 'undefined') {
-    const token = localStorage.getItem(google_authorization.key);
+const is_authorized = () =>
+  typeof window !== "undefined" &&
+  localStorage.getItem(google_authorization.key);
 
-    if (token) {
-      return true;
-    }
-  }
-  return false;
-}
-
-export const logout = () => {
-  if (typeof window !== 'undefined') {
+const logout = () => {
+  if (typeof window !== "undefined") {
     localStorage.removeItem(google_authorization.key);
     Router.push("/login");
   }
-}
+};
+
+export { google_authorization, is_authorized, logout };
```

Comment:
google_authorization is being exported, and used in pages/login.js.

1. should remove google_authorization from the export here
2. define a function in constants/authorization.js
`const authorize = (token) => {
    localStorage.setItem(google_authorization.key, token);
}
`
3. import this new function to be used /pages/login.js@line 20

Reason: we shouldn't allow google_authorization to be available to anything outside of /constants/authorization.js